cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -87,7 +93,7 @@ async fn run_docker_command(cmd: &Cmd) -> Result<(), Error> {
         ..Default::default()
     };
 
-    let container_name = format!("stellar-{}", cmd.network);
+    let container_name = get_container_name(cmd);
```

Comment:
Just realized that this function takes the `self` as the only arg. So it seems like it should be in an `impl Cmd`.

Reply:
After reverting the change mentioned in https://github.com/stellar/stellar-cli/pull/1423#discussion_r1678472001 `get_container_name` is back implemented on Args with an optional network passed in